lib/at_cmd_parser/at_cmd_parser.c

```diff
@@ -249,12 +249,12 @@ static int at_parse_process_element(const char **str, int index,
 		tmpstr++;
 	} else if (state == NUMBER) {
 		char *next;
-		int value = (uint32_t)strtoul(tmpstr, &next, 10);
+		int32_t value = (int32_t)strtoul(tmpstr, &next, 10);
```

Comment:
Shouldn't `strtol` be used instead then?

Reply:
Yes, overlooked that detail.